client/jetpack-cloud/components/layout/style.scss

```diff
@@ -89,7 +89,7 @@
 }
 
 .jetpack-cloud-layout__header.has-actions > .jetpack-cloud-layout__header-main {
-	@include breakpoint-deprecated( "<1280px" ) {
+	@include breakpoint-deprecated( "660px-1280px" ) {
```

Comment:
Minor fix to not add the extra margin on small screen devices since the actions are not shown.